docs/user-guide/concepts.md

```diff
@@ -280,8 +280,7 @@ their members.
 To implement hierarchical quotas, HNC automatically creates `ResourceQuota` objects in each
 affected namespace. This is a part of the internal implementation and shouldn't be modified or
 inspected. Use the `kubectl hns hrq` command to inspect hierarchical quotas,
```

Comment:
@zfrhv Could you please add `kubectl get hrq` here too?

Reply:
Done
Tell me if you like it

Reply:
@zfrhv 
Sorry, my explanation was inadequate...
I intended tha following fix:

> Use the `kubectl hns hrq` command to inspect hierarchical quotas,

->

Use the `kubectl hns hrq` or `kubectl get hrq` command to inspect hierarchical quotas,

Reply:
what about now?

Reply:
How about fix like that?

https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/163a06afac9978823299425218898f45a34c4946/docs/user-guide/concepts.md?plain=1#L280-L294

-> 

Remove (As written, duplicates quickstart example)

https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/163a06afac9978823299425218898f45a34c4946/docs/user-guide/concepts.md?plain=1#L297

-> 

> Use the `kubectl hns hrq` or `kubectl get hrq` command to inspect hierarchical quotas,

Reply:
what about now?

Reply:
Thanks @zfrhv !
I think it's good.
Let's have @adrianludwin check it out.